### DISCUSSION TITLE
Set a session name in a layout

### DISCUSSION THREAD

**Discussion by darklajid** [zellij-org/zellij#2694] [Q&A]
I have a "I need to get stuff done" layout and would like the window to show something like "zellij(work)" instead of - an actual example from right now - "zellij(wonderful-pig)".

Is there a way to fix this in the config or the layout files?

**Comment by Lockszmith-GH** [comment]
`--session-name <SESSION_NAME>`, see https://zellij.dev/documentation/command-line-options

I couldn't [find anything ](https://zellij.dev/documentation/command-line-options?search=session%20name) specific for the layout configuration - although I suspect the scope of the session is different from the layout.
Note that you _can_ name a tab or a pane.

**Reply by darklajid** [reply]
Yup, was aware of tab/pane names, this is just about the terminal window title for me and just an itch to scratch. If it isn't possible in the layout then I'll just live with it or .. set up an alias in my shell maybe
